keycloak-user-api/src/main/java/io/enmasse/user/keycloak/KeycloakUserApi.java

```diff
@@ -244,6 +234,49 @@ private void applyAuthorizationRules(RealmResource realm, User user, UserResourc
         }
     }
 
+    /**
+     * Create the set of desired groups.
+     * 
+     * @param user The user to create the groups for.
+     * @return A set of groups.
+     */
+    static Set<String> createDesiredGroupsSet(final List<UserAuthorization> authorization) {
+
+        final Set<String> desiredGroups = new HashSet<>();
+
+        for (UserAuthorization userAuthorization : authorization) {
+
+            for (Operation operation : userAuthorization.getOperations()) {
+
+                if (userAuthorization.getAddresses() == null || userAuthorization.getAddresses().isEmpty()) {
+
+                    switch (operation) {
+                    case manage:
```

Comment:
could you please format this switch?

Reply:
Sure, if you can point me to the formatter rules 😁

Reply:
😁 https://google.github.io/styleguide/javaguide.html#s4.8.4-switch

Reply:
IntelliJ default format 😅👍

Reply:
I am sorry, but I can only handle Eclipse formats :wink:

Reply:
I found a ready-to-use Eclipse file for using the Google code style for Java. I will go for that then.

Reply:
I don't think you are using the Google code style, applying it causes a whole bunch of changes.

Reply:
I think the closest matching is oracle java conventions or something, if there is such a formal specification

Reply:
I think I got it. Looks like a modifed version of Google's code formatter, with 4 spaces indentation instead of 2.

I pushed a change, please have a look.

Reply:
LGTM!